development/kops/set_environment.sh

```diff
@@ -61,8 +61,7 @@ fi
 export PREFLIGHT_CHECK_PASSED
 
 export KUBERNETES_VERSION=$(cat ../../projects/kubernetes/kubernetes/${RELEASE_BRANCH}/GIT_TAG)
-export DEFAULT_REPOSITORY_URI=public.ecr.aws/eks-distro
-export REPOSITORY_URI=${REPOSITORY_URI:-${DEFAULT_REPOSITORY_URI}}
+export REPOSITORY_URI=${REPOSITORY_URI:-public.ecr.aws/h1r8a7l5}
```

Comment:
Sorry, I wasn't being clear. We *don't* want the dev build to be default here. By default, users would want the regular ecr repo

Reply:
```suggestion
export REPOSITORY_URI=${REPOSITORY_URI:-public.ecr.aws/eks-distro}
```